src/core/openthread-core-default-config.h

```diff
@@ -559,4 +559,34 @@
 #define OPENTHREAD_CONFIG_PLATFORM_ASSERT_MANAGEMENT            0
 #endif
 
+/**
```

Comment:
To keep with convention of this file, these new options should be named OPENTHREAD_CONFIG_...